nilearn/input_data/nifti_labels_masker.py

```diff
@@ -181,7 +191,7 @@ def _check_fitted(self):
                              'You must call fit() before calling transform().'
                              % self.__class__.__name__)
 
-    def transform(self, imgs, confounds=None):
+    def transform_single_imgs(self, imgs, confounds=None):
         """Extract signals from images.
 
```

Comment:
I made the doc more precise and moved it into BaseMasker so that it's propagated to all subclasses.